stories/rating-indicator/rating-indicator.stories.js

```diff
@@ -0,0 +1,510 @@
+            
+            <input aria-label="4 star" type="radio" class="fd-rating-indicator__input" id="rating-disabled-4" name="rating-disabled" value=4" disabled>
+            <label class="fd-rating-indicator__label" for="rating-disabled-4"></label>
+            
+            <input aria-label="5 star" type="radio" class="fd-rating-indicator__input" id="rating-disabled-5" name="rating-disabled" value="5" disabled>
+            <label class="fd-rating-indicator__label" for="rating-disabled-5"></label>
+        </div>
+        <span class="fd-rating-indicator__dynamic-text">(2 of 5)</span>
+    </div>
+</div>
+`;
+
+Disabled.parameters = {
+    docs: {
+        storyDescription: `
+To disable the \`Rating Indicator\` component needs to be added attribute \`[disable]\` 
+to all inputs, also needs to add one of these (\`.is-disable\`, \`aria-disable="true"\`, \`[disable]\`) 
+selectors to the \`fd-rating-indicator\` element.
+        `
+    }
+};
+
+export const DisplayMode = () => `    
+    <div class="example-container">
+        <div class="fd-rating-indicator is-display-mode">
+            <div class="fd-rating-indicator__container" aria-label="Star Rating (out of 5)">
+                <input aria-label="1 star" type="radio" class="fd-rating-indicator__input" id="rating-display-mode-1" name="rating-display-mode" value="1" disabled>  
+                <label class="fd-rating-indicator__label" for="rating-display-mode-1"></label>
+                
+                <input aria-label="2 star" type="radio" class="fd-rating-indicator__input" id="rating-display-mode-2" name="rating-display-mode" value="2" disabled checked>
+                <label class="fd-rating-indicator__label" for="rating-display-mode-2"></label>
+                
+                <input aria-label="3 star" type="radio" class="fd-rating-indicator__input" id="rating-display-mode-3" name="rating-display-mode" value="3" disabled>
+                <label class="fd-rating-indicator__label" for="rating-display-mode-3"></label>
+                
+                <input aria-label="4 star" type="radio" class="fd-rating-indicator__input" id="rating-display-mode-4" name="rating-display-mode" value=4" disabled>
+                <label class="fd-rating-indicator__label" for="rating-display-mode-4"></label>
+                
+                <input aria-label="5 star" type="radio" class="fd-rating-indicator__input" id="rating-display-mode-5" name="rating-display-mode" value="5" disabled>
+                <label class="fd-rating-indicator__label" for="rating-display-mode-5"></label>
+            </div>
+            <span class="fd-rating-indicator__dynamic-text">(2 of 5)</span>
+        </div>
+    </div>
+`;
+
+DisplayMode.parameters = {
+    docs: {
+        storyDescription: `
+To make the \`Rating Indicator\` component display mode needs to be added class \`.is-display-mode\` to the \`fd-rating-indicator\` element.
```

Comment:
"Data can be presented as label-value field pairs without editable fields in display-only forms, see **Form Grid** for more details. If you want to include a rating indicator in a display-only form, add the `.is-display-mode` to the `fd-rating-indicator` element."

Reply:
It would be great to link **Form Grid** to its corresponding page.

Reply:
fixed